app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  has_many :comment, counter_cache: :post_counter
+  belongs_to :author
+  has_many :like
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :comment_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :title, presence: true, length: { maximum: 250 }
+
```

Comment:
- [ Required ] I think it would be better if you use [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#increment_post_counter` method every time a new post has been saved to database, then make the `increment_post_counter` method private. Please see the example below and rectify this.

## Example code 
```ruby
  after_save :increment_post_counter

  private

  def increment_post_counter
    author.increment!(:post_counter)
  end
```